design-patterns/gc-intro/_base.scss

```diff
@@ -0,0 +1,5 @@
+/* REMOVE ONCE IMPLEMENTED IN WET-BOEW */
+.bg-half-width-right {
```

Comment:
As I proposed earlier

```suggestion
.bg-half-right {
```

And let also add the `bg-half-left` to have a complete utilities suite.

Reply:
Not needed anymore.